module_1/first-react-app/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import SearchBox from './components/SearchBox.js';
+import MovieGenre from './components/MovieGenre.js';
+import CounterLegacy from './components/CounterLegacy.js';
+
+
+
+
+
```

Comment:
Please remove the blank lines

Reply:
Done in commit: c7a4e1e3bff35922c071c59fe3905a3da37fefef